pkg/smartcontract/context/item.go

```diff
@@ -36,7 +36,7 @@ func (it *Item) AddSignature(pub *keys.PublicKey, sig []byte) {
 // MarshalJSON implements json.Marshaler interface.
 func (it Item) MarshalJSON() ([]byte, error) {
 	ci := itemAux{
-		Script:     it.Script,
+		Script:     it.Script.Reverse(), // script should be marshaled in BE but default marshaler uses LE.
```

Comment:
It'd be nice to check this against some C# examples.

Reply:
They store `Script` as []byte in LE, so fixed